distribution/changelog.txt

```diff
@@ -19,6 +19,7 @@
 - Feature: [#13614] Add terrain surfaces from RollerCoaster Tycoon 1.
 - Feature: [#13675] [Plugin] Add context.setInterval and context.setTimeout.
 - Change: [#13346] [Plugin] Renamed FootpathScenery to FootpathAddition, fix typos.
+- Fix: [#4605, #11912] Water palettes not updating properly when selected in Object Selection.
```

Comment:
Nitpic, but we don't use the present continuous.

```suggestion
- Fix: [#4605, #11912] Water palettes are not updated properly when selected in Object Selection.
```

On the subject of nitpicking: could you omit the dot at the end of the commit message? (But keep the one in the changelog!)

Reply:
Nitpicks fixed. 😉